CMakeLists.txt

```diff
@@ -83,6 +83,37 @@ if( CASA_BUILD )
    if( NOT DATA_DIR )
       set(DATA_DIR "%CASAROOT%/data")
    endif( )
+   set(USE_FFTW3 ON)
+   set(USE_THREADS ON)
+   ###
+   ### after CASA switches from ASAP to libsakura, CASA will no longer
+   ### use casacore's python module and it will no longer use boost...
+   ###
+   set(BUILD_PYTHON ON)
+   set(Boost_NO_BOOST_CMAKE 1)
+   if (EXISTS "/opt/casa/01/include/python2.7")
+      set(ENV{PATH} "/opt/casa/01/bin" $ENV{PATH})
+      ### RHEL7
+      set(PYTHON_LIBRARY "/opt/casa/01/lib/libpython2.7.so")
+      set(PYTHON_INCLUDE_DIR "/opt/casa/01/include/python2.7")
+      set(PYTHON_EXECUTABLE:FILEPATH "/opt/casa/01/bin/python")
+      if (EXISTS "/usr/include/boost")
+         set(BOOST_ROOT "/usr")
+      endif( )
+   elseif(EXISTS "/usr/lib64/casa/01/include/python2.7")
+      ### RHEL5/RHEL6
+      set(ENV{PATH} "/usr/lib64/casa/01/bin" $ENV{PATH})
```

Comment:
I was trying to have cmake find the right exectuables as part of it's configuration phase but did not want users to have to configure their path outside of cmake... I am not completely sure that this did achieve that goal, but this was the reference I used:

http://stackoverflow.com/questions/21047399/cmake-set-environment-variables-from-a-script